src/main/java/com/moodybluez/enterprise/dao/IDateDAO.java

```diff
@@ -0,0 +1,13 @@
+package com.moodybluez.enterprise.dao;
+
+import com.moodybluez.enterprise.dto.Date;
+
+public interface IDateDAO {
+
+    boolean createEntry(Date date);
+
+    Date fetchByWeekDay(int weekDayID);
+
+    Date fetchByDate(String date);
```

Comment:
Added Persistence level methods (fully implemented with logic in actual classes) to this layer, moved from service